tsdb/head.go

```diff
@@ -1167,6 +1179,7 @@ func (h *Head) gc() {
 			values[t.Name] = ss
 		}
 		ss.set(t.Value)
+		h.metrics.labelpairValues.Set(float64(ss.Size()))
```

Comment:
This isn't right, this is the number of label values for the last label touched.